LoRaEngine/modules/LoRaBaseStationModule/Dockerfile.amd64

```
@@ -0,0 +1,17 @@
+#This docker builds a container for the LoRa Base station
+
+FROM debian as build
+RUN apt-get update
+RUN apt-get install -y git
+RUN apt-get install -y --no-install-recommends apt-utils build-essential
+RUN git clone https://github.com/lorabasics/basicstation.git
```

How about also cloning [the latest release know at this time](https://github.com/lorabasics/basicstation/releases/tag/v2.0.5) so we are always sure it's that version?

```suggestion
RUN git clone --branch v2.0.5 --single-branch --depth 1 https://github.com/lorabasics/basicstation.git
```

---

good idea!